test/clojure/core_test/when_first.cljc

```diff
@@ -0,0 +1,29 @@
+(ns clojure.core-test.when-first
+  (:require [clojure.test :as t :refer [deftest testing is are]]
+            [clojure.core-test.portability #?(:cljs :refer-macros :default :refer) [when-var-exists]]))
+
+
+(when-var-exists when-first
+  (deftest test-when-first
+    ;; `testing` sections are optional, depending on how you want to
+    ;; structure your tests. If you have a lot of tests and they group
+    ;; together in subgroups, then use `testing`. The `testing` form
+    ;; can also be a nice way to group tests that only apply to a
+    ;; subset of Clojure implementations. These can then be guarded by
+    ;; reader conditionals.
```

Comment:
Could we remove this? It's more for advice for the writer than the readers of finished code

Reply:
Sorry didn't realize this was draft!